scipy/stats/stats.py

```diff
@@ -5718,3 +5720,175 @@ def rankdata(a, method='average'):
 
     # average method
     return .5 * (count[dense] + count[dense - 1] + 1)
+
+
+def ratio_unif(f, size=1, x1=None, x2=None, y2=None, shift=0, maxiter=None,
+               bounds=None):
+    """
+    Generate random variables using ratio-of-uniforms method.
+
+    Given a continuous univariate density f and a constant c, define the set
```

Comment:
If this is going to be part of the public API then I think the language needs simplifying significantly.
Does `f` represent the PDF? I would use PDF instead because all the stats distributions have a PDF method.